docs/02_getting-started/03_smart-contract-development/04_data-persistence.md

```diff
@@ -68,9 +68,9 @@ struct person {
 ```
 Great. The basic data structure is now complete.
 
-Next, define a `primary_key` method. Every multi_index struct requires a *primary key* to be set. Behind the scenes, this method is used according to the index specification of your multi_index instantiation. EOSIO wraps [boost::multi_index](https://www.boost.org/doc/libs/1_59_0/libs/multi_index/doc/index.html)
+Next, define a `primary_key` method. Every multi_index struct requires a *primary key* method. Behind the scenes, this method is used according to the index specification of your multi_index instantiation. EOSIO `multi_index` wraps [boost::multi_index](https://www.boost.org/doc/libs/1_59_0/libs/multi_index/doc/index.html)
```

Comment:
`multi_index` wrapped around ticks, we shall use it like this everywhere, or not use it surrounded by ticks, whatever it is we should be consistent.

Reply:
I think so but I will merge this PR first